package.json

```diff
@@ -99,35 +41,30 @@
 		"eslint-plugin-jsx-a11y": "^6.1.1",
 		"eslint-plugin-prettier": "^2.6.2",
 		"eslint-plugin-react": "^7.11.1",
-		"favicons-webpack-plugin": "^0.0.9",
 		"file-loader": "^2.0.0",
 		"flow-bin": "^0.81.0",
 		"fs-extra": "^7.0.1",
-		"html-webpack-harddisk-plugin": "^1.0.1",
-		"html-webpack-plugin": "^3.2.0",
 		"husky": "^1.3.1",
-		"jest": "^23.6.0",
+		"jest": "^24.9.0",
 		"jest-emotion": "^9.2.7",
 		"lint-staged": "^7.2.2",
 		"prettier": "^1.14.2",
-		"style-loader": "^0.23.0",
-		"webpack": "^4.19.0",
-		"webpack-bundle-analyzer": "^3.0.2",
-		"webpack-cli": "^3.1.0",
-		"webpack-dev-server": "^3.1.8",
-		"write-file-webpack-plugin": "^4.5.0"
+		"style-loader": "^0.23.0"
 	},
-	"authors": [
-		"Divyanshu Maithani <div.blackcat@gmail.com> (https://github.com/divyanshu013)",
-		"Deepak Grover <deepakgrover333@gmail.com> (https://github.com/metagrover)",
-		"Lakhan Samani <lakhan.m.samani@gmail.com> (https://github.com/lakhansamani",
-		"Appbase.io <info@appbase.io> (https://github.com/appbaseio)"
-	],
-	"jest": {
-		"testPathIgnorePatterns": [
-			"<rootDir>/app/src/batteries",
-			"<rootDir>/node_modules/"
-		]
+	"license": "MIT",
+	"scripts": {
+		"dev:browser": "lerna run watch --scope dejavu-data-browser --stream",
+		"dev:dejavu": "lerna run watch --scope dejavu-main --stream",
+		"dev:dejavu:site": "lerna run watch:site --scope dejavu-main --stream",
+		"build:browser": "lerna run build --scope dejavu-data-browser --stream",
+		"build:dejavu": "yarn build:browser && lerna run build --scope dejavu-main --stream",
+		"build:dejavu:app": "yarn build:browser && lerna run build:app --scope dejavu-main --stream",
+		"build:dejavu:site": "yarn build:browser && lerna run build:site --scope dejavu-main --stream"
```

Comment:
Can you add code formating using `prettier`

https://github.com/appbaseio/dejavu/blob/dev/package.json#L23

Also, chrome extension build is missing
https://github.com/appbaseio/dejavu/blob/dev/package.json#L24

Make sure extension & its build works correctly